### DISCUSSION TITLE
Different output from HF and Tensorrt-llm

### DISCUSSION THREAD

**Discussion by Ericoool9614** [NVIDIA/TensorRT-LLM#2754] [Q&A]
Model: Internvl2-8B
Precision: BF16, no quantization
No sampling strategy (temperature=0, do_sample=False in HF generation_config, so both are greedy search) 
Single GPU execution, no model parallelism
No batch (batch size = 1)
Inference is performed using Hugging Face model.chat method and Tensorrt-llm MultimodalModelRunner.run() method.

**Comment by Ericoool9614** [comment]
The difference appears after the first logits output from runtime.generation.handle_per_step
